JDK9 Module System to read system classes instead of rt.jar

bilalekremharmansa opened the pull request:
+ModuleSystem.java

bilalekremharmansa committed:
SSS supports JDK 9

bilalekremharmansa committed:
Chooser.java reverted back

bilalekremharmansa committed:
ModuleSystem.java, changed System.out.print() as System.err.print()

maeyler commented:
GitKraken accepted these changes to the correct branch

No need to merge the PR any more

maeyler commented:
açık bırakalım bakalım  nasıl olacak

maeyler committed:
Turkish chars modified

unicode was not used

maeyler committed:
Chooser reads modules in Java 9

reflection is used in order to compile under earliear versions

maeyler committed:
Chooser reads modules in Java 9

A tiny modification
First problem is solved

maeyler committed:
V2.08a

No need to scale fonts in Java 9
Swing components are automatically scaled
http://openjdk.java.net/jeps/263

maeyler committed:
ClassSummary modified

Class file format modified in Java 7 and 9
https://docs.oracle.com/javase/specs/jvms/se9/html/jvms-4.html

bilalekremharmansa committed:
UndoManager modified

undoableEditHappened() modified on Java 9. SSS expects an DocumentEvent object when the method is invoked. However, DocumentEvent is wrapped by DefaultDocumentEventUndoableWrapper. UndoManager uses Reflection to get DocumentEvent inside of wrapper class.

https://netbeans.org/bugzilla/show_bug.cgi?id=270142#c3

maeyler commented:
UndoManager modified:  Wonderful fix -- thank you

maeyler committed:
Little changes

maeyler committed:
Use compiler in javax.tools

version strings unified

maeyler committed:
V2.08

supports Java 9
works under Java 6 or later

maeyler commented:
_Sağol Bilal, güzel ve uyumlu oldu_
